src/web/signer.ts

```diff
@@ -16,27 +16,34 @@
 import {
   ArconnectSigner,
   ArweaveSigner,
-  DataItem,
+  DataItemCreateOptions,
   EthereumSigner,
   HexSolanaSigner,
   InjectedEthereumSigner,
+  Signer,
   createData,
+  deepHash,
+  stringToBuffer,
 } from '@dha-team/arbundles';
 
+import { TurboEventEmitter } from '../common/events.js';
 import { TurboDataItemAbstractSigner } from '../common/signer.js';
 import {
-  StreamSizeFactory,
   TurboDataItemSignerParams,
+  TurboSignedDataItemFactory,
   TurboSignedRequestHeaders,
   WebTurboFileFactory,
 } from '../types.js';
-import { readableStreamToBuffer } from '../utils/readableStream.js';
+import { createUint8ArrayReadableStreamFactory } from '../utils/readableStream.js';
 
 /**
  * Utility exports to avoid clients having to install arbundles
  */
 export { ArconnectSigner, ArweaveSigner, EthereumSigner, HexSolanaSigner };
 
+export type DeepHashChunk = Uint8Array | AsyncIterable<Buffer> | DeepHashChunks;
+export type DeepHashChunks = DeepHashChunk[];
```

Comment:
why are these necessary to be exported?

Reply:
https://github.com/ardriveapp/turbo-sdk/pull/264/commits/e56a666a8980d6eb9ea4c110e3d909aa3d2a0c0c

Its not necessary